common/static/common/js/spec_helpers/page_helpers.js

```diff
@@ -1,18 +1,38 @@
-define([ // jshint ignore:line
-],
-function() {
-    'use strict';
-    var getLocationHash;
-    /**
-     * Helper method that returns url hash.
-     * @return {String} Returns anchor part of current url.
-     */
-    getLocationHash = function() {
-        return window.location.hash;
-    };
+define(["backbone"],
+    function(Backbone) {
+        'use strict';
+        var getLocationHash, installMockBackboneHistory;
 
-    return {
-        'getLocationHash': getLocationHash
-    };
+        /**
+         * Helper method that returns url hash.
+         * @return {String} Returns anchor part of current url.
+         */
+        getLocationHash = function() {
+            return window.location.hash;
+        };
 
-});
+        /**
+         * Mocks the Backbone history so that navigation doesn't change
```

Comment:
Add more detailed comments. In particular:

1) The Backbone views will still update because the callbacks occur. But...
2) The URL as displayed in the browser will not change-- this ensures that URLs do not change when the tests run.